webclient/app/src/app/commons/statement/Statement.jsx

```diff
@@ -36,7 +43,7 @@ Statement.propTypes = {
 
 Statement.defaultProps = {
     message: "Keine Nachricht festgelegt",
-    icon: <CheckBoxOutlineBlank/>,
+    icon: <img src={iconEmpty} alt="empty"/>
```

Comment:
Why did you change that icon? It should be an icon, that will be replaced later on. Also, I think that the icons should look familiar to each other and not implement a bunch of styles. I would recommend staying to the standard google icons pack

Reply:
I didn't find any suitable icons in the standard google icon pack. This should be changed later on, but the empty checkbox is missleading.